### PR TITLE
Cloud Patches 1.0.9 Release

### PR DESCRIPTION
**Pull request by bdenham**
## Purpose of this pull request

This pull request (PR) updates Cloud Patches release notes for version 1.0.9.

## Preview from local build

image

## Affected DevDocs pages

https://devdocs.magento.com/cloud/release-notes/mcp-release-notes.html

whatsnew
 Cloud Patches 1.0.9 [release notes](https://devdocs.magento.com/cloud/release-notes/mcp-release-notes.html).

### CODE REVIEW

**Review by hguthrie — APPROVED** [review]
"Silently failed" 😲 
Nice.

File: src/cloud/release-notes/mcp-release-notes.md [hunk 1]
```diff
@@ -21,6 +21,14 @@ See [Apply patches]({{site.baseurl}}/cloud/project/project-patch.html) for instr
...
 <!--Add release notes below-->
 
+## v1.0.9
+*Release date: February 1, 2021*<br/>
+
+-  {:.fix} **Compatibility with PHP 7.4**—Fixed a compatibility issue with PHP 7.4.
+-  {:.fix} **Deprecated patches become visible**—Fixed issue in which deprecated patches become visible is the patches table after applying a replacement patch that contains the entire contents of the deprecated patch. This could happen if you applied a patch that combined several other patches.<!--MC-40626-->
+-  {:.fix} **Layered navigation**—Fixed the issue that prevented the layered navigation from showing all options when the B2B Shared Catalog was enabled.<!--MCLOUD-6923-->
+-  {:.fix} **Silent failures when applying patches**—Fixed an issue in which the `git apply` command silently failed to apply patches in some environments.<!--MC-40529-->
+
```
[the remark this answers is not in the document]
Comment (hguthrie) [reply src/cloud/release-notes/mcp-release-notes.md:31]: Looks like the fixes to `magento-cloud-patches` can be listed with `{:.fix}` bullet and the new patch should be called out by name in bold with description following, as in previous release notes. That seems to be a readable way.

[the remark this answers is not in the document]
Comment (bdenham) [reply src/cloud/release-notes/mcp-release-notes.md:31]: Yep. I get it now. My first patch job. Thanks for the guidance!

**Comment by bdenham** [comment]
running tests

**Comment by bdenham** [comment]
running tests

**Pull request merged**